git/github.go

```diff
@@ -0,0 +1,46 @@
+package git
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/cdnjs/tools/util"
+)
+
+// Stars holds the number of stars for a GitHub repository.
+type Stars struct {
+	Stars uint `json:"stargazers_count"`
+}
+
+// Gets the protocol, either http or https.
+func getProtocol() string {
```

Comment:
could we maybe shared this npm code?

Reply:
Like this? or is there a better way to do it?